l1-contracts/src/core/libraries/ConstantsGen.sol

```diff
@@ -18,7 +18,7 @@ library Constants {
   uint256 internal constant ARGS_LENGTH = 16;
   uint256 internal constant MAX_NOTE_HASHES_PER_CALL = 16;
   uint256 internal constant MAX_NULLIFIERS_PER_CALL = 16;
-  uint256 internal constant MAX_PRIVATE_CALL_STACK_LENGTH_PER_CALL = 4;
+  uint256 internal constant MAX_PRIVATE_CALL_STACK_LENGTH_PER_CALL = 5;
```

Comment:
I had to increase this limit since adding liquidity involves:
 - token0.transfer_private_to_public (max amount)
 - token0.prepare_balance_increase (change)
 - token1.transfer_private_to_public (max amount)
 - token1.prepare_balance_increase (change)
 - liquidity_token.prepare_balance_increase (minted)

Implementing https://github.com/AztecProtocol/aztec-packages/issues/10286 would reduce this to three calls.